verifier/src/identity.rs

```diff
@@ -609,5 +771,49 @@ mod test {
         let verifier = TrustedIdentitiesVerifier::new(allowed_identities);
         let verification = verifier.verify(&identity);
         assert_eq!(verification.is_success().unwrap_u8(), 0);
+
+        let displayable = VerificationTreeDisplay::new(&verifier, verification);
+        let expected = r#"
+            - [ ] No enclave identity matched for:
+              - MRENCLAVE: 0101010101010101010101010101010101010101010101010101010101010101
+              - MRSIGNER key hash: 0202020202020202020202020202020202020202020202020202020202020202
+              - ISV product ID: 3
+              - ISV SVN: 4
+              - advisories: IDs: {"four", "one", "three", "two"} Status: ConfigurationAndSWHardeningNeeded
+              Searched through the following identities:
+              - [ ] Both of the following must be true:
+                - [ ] The MRENCLAVE should be 0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b, but the actual MRENCLAVE was 0101010101010101010101010101010101010101010101010101010101010101
+                - [x] The allowed advisories are IDs: {"four", "one", "three", "two"} Status: ConfigurationAndSWHardeningNeeded
+              - [ ] Both of the following must be true:
```

Comment:
Up for suggestions on better ways to separate out the individual identity attempts, at the same time this is probably good enough.